### DISCUSSION TITLE
Are custom aliases a thing?

### DISCUSSION THREAD

**Discussion by adamlogic** [rcoh/angle-grinder#202] [Q&A] [answered]
I see the "aliases" section in the README, but it's unclear if there's a way to have custom aliases in a local config directory or something like that. I have some [frequent queries I run](https://judoscale.com/guides/heroku-logs-tools) that aren't generally useful for others, but I'd love to package them for myself in a way so I don't have to continually copy & paste.

**Comment by rcoh** [comment]
suggested aliases is an open area—I think we should add either a standard directory or environment variable that gets searched for aliases. Happy to mentor / guide if you're interested in contributing

**Reply by adamlogic** [reply]
Thanks for the answer and the offer! Since I've never touched Rust in my life, that might be biting off a bit more than I have the bandwidth for.

I have no expectation for you to build my feature request, though. I just wanted to make sure I wasn't missing something.

**Reply by rcoh** [reply, marked as answer]
This is done now!
